test/unit/menu-test.js

```diff
@@ -80,7 +81,7 @@ describe('Menu', () => {
     });
 });
 
-describe('Settings Menu', () => {
+describe.only('Settings Menu', () => {
```

Comment:
remove this! -- can we add a lint rule for this?

Reply:
We can. For reference see https://github.com/jwplayer/hls.js/commit/76e609f1bb5e4f1a6b2ad150be2085ce9d9245b9